.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: node_js
 node_js:
-  - "node" # latest stable Node.js release
```

Comment:
yeah it would lead to unexpected failures, node major versions are still breaking versions. We are experimenting with Greenkeeper sending PRs to update the these versions, so as soon as version 9 comes out it will send a PR updating the `.travis.yml` file. I think that’d be the perfect way